pkg/gui/confirmation_panel.go

```diff
@@ -37,19 +37,25 @@ func (gui *Gui) closeConfirmationPrompt(g *gocui.Gui) error {
 	return g.DeleteView("confirmation")
 }
 
-func (gui *Gui) getMessageHeight(message string, width int) int {
+func (gui *Gui) getMessageHeight(v *gocui.View, message string, width int) int {
 	lines := strings.Split(message, "\n")
 	lineCount := 0
-	for _, line := range lines {
-		lineCount += len(line)/width + 1
+	// if we need to wrap, calculate height to fit content within view's width
+	if v.Wrap {
+		for _, line := range lines {
+			lineCount += len(line)/width + 1
+		}
+	} else {
+		lineCount = len(lines)
 	}
 	return lineCount
 }
 
 func (gui *Gui) getConfirmationPanelDimensions(g *gocui.Gui, prompt string) (int, int, int, int) {
 	width, height := g.Size()
 	panelWidth := width / 2
-	panelHeight := gui.getMessageHeight(prompt, panelWidth)
+	view := g.CurrentView()
```

Comment:
sorry for the late reply, I've just gotten back from my holiday :)

thanks for making this PR!

I think we should explicitly refer to the confirmation panel view here in case something else momentarily has focus:
```suggestion
	view, err := gui.g.View("confirmation")
	if err != nil { // confirmation panel was deleted so we just return empty values
		return 0,0,0,0 
	}
```

Reply:
nice point, LGTM 🍕